src/commands/synthetics/compatibility.ts

```diff
@@ -24,3 +24,18 @@ export const replaceGlobalWithDefaultTestOverrides = (
 
   return config
 }
+
+export const replaceConfigWithTestOverrides = (
+  config?: UserConfigOverride,
+  testOverrides?: UserConfigOverride
+): UserConfigOverride => {
+  const isConfigUsed = Object.keys(config ?? {}).length !== 0
+  const isTestOverridesUsed = Object.keys(testOverrides ?? {}).length !== 0
+
+  // If both config and testOverrides exist use testOverrides
+  if (isConfigUsed && !isTestOverridesUsed) {
+    return {...config}
```

Comment:
No need to do a copy, and it hides the function's purpose

```suggestion
    return config
```

Reply:
Is there a reason not to make a copy? This way i feel like i separate more config from testOverrides. If config gets changed in some weird way it won't impact testOverrides 

Also what do you mean by it hides the function's purpose 🤔

Reply:
okay i see 🤔